package.json

```diff
@@ -194,8 +194,9 @@
     "lint-code": "eslint \"{example,features,scripts,src,test}/**/*.js\"",
     "lint-autofix": "eslint --fix \"{example,features,scripts,src,test}/**/*.js\"",
     "lint": "yarn run lint-code && yarn run lint-dependencies",
-    "prefeature-test": "yarn run build-local",
-    "prepublish": "yarn run build-local",
+    "postinstall": "node scripts/build-local.js",
+    "prefeature-test": "node scripts/build-local.js",
+    "prepublish": "node scripts/build-local.js",
```

Comment:
prepublish and prefeature-test should remain as they were. The lib folder needs to be updated even it if exists in order to ensure it contains the latest code